### DISCUSSION TITLE
PingVin update failed

### DISCUSSION THREAD

**Discussion by Narkowii** [tteck/Proxmox#3608] [Help Needed] [answered]
Hello
I have PingVin installed with the tteck script; but i have an update error

`   
 ✓ Stopped Pingvin Share
 | Updating Pingvin Share   
[ERROR] in line 65: exit code 0: while executing command git checkout $(git describe --tags `git rev-list --tags --max-count=1`) &> /dev/null

root@pingvin:~#`

**Comment by FieserOstfriese** [comment]
Same Error by me

**Comment by tteck** [comment]
https://stonith404.github.io/pingvin-share/setup/upgrading#stand-alone

**Comment by Rems-38600** [comment, marked as answer]
> Hello I have PingVin installed with the tteck script; but i have an update error
> 
> ` ✓ Stopped Pingvin Share | Updating Pingvin Share [ERROR] in line 65: exit code 0: while executing command git checkout $(git describe --tags`git rev-list --tags --max-count=1`) &> /dev/null
> 
> root@pingvin:~#`

Bonjour,
J'ai mise à jour la version 1.0.2 vers la version 1.1.3 en faisant la manipulation suivante:
cd /opt/pingvin-share
Puis
git reset --hard
et enfin la commande update a fonctionner
 ✓ Stopped Pingvin Share
 ✓ Updated Pingvin Share
 ✓ Started Pingvin Share
 ✓ Updated Successfully
root@pingvin:/opt/pingvin-share# 

Good morning,
I updated version 1.0.2 to version 1.1.3 by doing the following:
cd /opt/pingvin-share
Then
git reset --hard
and finally
update
The command worked
 ✓ Stopped Pingvin Share
 ✓ Updated Pingvin Share
 ✓ Started Pingvin Share
 ✓ Updated Successfully
root@pingvin:/opt/pingvin-share#
